src/ONE.Models.CSharp/ONE.Models.CSharp.csproj

```diff
@@ -2,9 +2,9 @@
 
   <PropertyGroup>
     <!--For Release packages update the VersionPrefix and remove any VersionSuffix-->
-    <VersionPrefix>4.37.0</VersionPrefix>
+    <VersionPrefix>4.38.0</VersionPrefix>
     <!--For Feature packages update the VersionSuffix with the jira ticket number-->
-    <VersionSuffix></VersionSuffix>
+    <VersionSuffix>AQI-6199-06</VersionSuffix>
```

Comment:
Since this is being merged into main do we need to remove this? I haven't made a protocol buffer change in a while

Reply:
Yes, the suffix should be removed before merging to main but I usually leave it in until I am ready to merge it.

Reply:
I usually remove that and merge to main after testing so I know that no further code changes are needed.